developing_packages/editable_packages.rst

```diff
@@ -153,14 +161,19 @@ This file can use the package reference to customize logic for a specific packag
 
    .. code-block:: ini
 
+       [build_folder]
+       build
+
+       [source_folder]
+       src
+
        [includedirs]
-       src/include
+       src
 
-       [cool/version@user/dev:includedirs]
-       src/core/include
+       [libdirs]
```

Comment:
We need to show in the docs the syntax with the `[reference:one-directory]`. What I meant by https://github.com/conan-io/examples/pull/47#discussion_r387561269 is that we should change the file in the example, not here in the docs. This syntax is quite confusing and having the example running with it will be useful

Reply:
Done, I've added to the jinja layout file

Reply:
But, before entering this snippet, the text says _"This file can use the package reference to customize logic for a specific package:"_, so it has to be here, or move the comment above.

Reply:
But, before entering this snippet, the text says "This file can use the package reference to customize logic for a specific package:", so it has to be here, or move the comment above.

Reply:
Indeed! Okay, it's updated now.